PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -413,7 +432,7 @@ def produceMET(self, process,  metType, metModuleSequence, postfix):
             process.load("PhysicsTools.PatUtils.patPFMETCorrections_cff")
             
         if postfix != "" and metType == "PF" and not hasattr(process, 'pat'+metType+'Met'+postfix):
-            noClonesTmp = [ "particleFlowDisplacedVertex", "pfCandidateToVertexAssociation" ]
+            noClonesTmp = [ "particleFlowDisplacedVertex", "pfCandidateToVertexAssociation" ] if not self._parameters["Puppi"].value else []
```

Comment:
@slava77 
For puppi this track association I think is also mis-leading as this one association is a bit different than the one used inside the charge hadrons. So prefer to have this disabled